packages/design-tokens/src/__tests__/selectors.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import styled from 'styled-components'
+import { render, RenderResult } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import 'jest-styled-components'
+
+import { selectors } from '..'
+
+const { bp } = selectors
```

Comment:
I understand it was a tricky decision to find a name that does not clash with existing naming. I wonder if `breakpoint` was originally the right name as in isolation I don't understand it although within the context of CSS it makes sense.

Maybe that opens the opportunity for another name?

Reply:
I figured that bp is a nice shorthand - which is somewhat a bit part of the objective of the selector anyway.

Upon releasing a major we could preserve the 'breakpoint' selector but use the new implementation.

I'm not sure what else to do to avoid a breaking change - it would be nice to get this out into the wild before the next major.

Very much open to suggestions on naming. 

Maybe something like `@media()`?

Reply:
We have agreed upon `mediaQuery` and `mq` for selectors associated with this new getter. 

`breakpoint` will be marked as deprecated.